We don't install the icons where upstream expects them.
so modify the code where needed.


Index: vlc-debian/modules/misc/notify/notify.c
===================================================================
--- vlc-debian.orig/modules/misc/notify/notify.c	2010-02-13 15:44:12.000000000 +0000
+++ vlc-debian/modules/misc/notify/notify.c	2010-02-13 15:44:27.000000000 +0000
@@ -231,14 +231,7 @@
     else /* else we show state-of-the art logo */
     {
         GError *p_error = NULL;
-        char *psz_pixbuf;
-        char *psz_data = config_GetDataDir( p_this );
-        if( asprintf( &psz_pixbuf, "%s/vlc48x48.png", psz_data ) >= 0 )
-        {
-            pix = gdk_pixbuf_new_from_file( psz_pixbuf, &p_error );
-            free( psz_pixbuf );
-        }
-        free( psz_data );
+        pix = gdk_pixbuf_new_from_file( "/usr/share/icons/hicolor/48x48/vlc.png", &p_error );
     }
 
     free( psz_arturl );
